lib/tasks/import.rake

```diff
@@ -32,4 +32,12 @@ namespace :import do
 
     puts whitehall_migration_url(whitehall_import.whitehall_migration_id, host: Plek.new.external_url_for("content-publisher"))
   end
+
+  desc "Import an array of documents from Whitehall Publisher using Whitehall's internal document ID e.g. import:whitehall_documents[\"123 456 789\"]"
```

Comment:
I think we should change "an array of" to "multiple", since the argument we're passing to the task is a string.

Reply:
Thanks - will fix